webpack/components/extensions/HostDetails/DetailsTabCards/RegistrationCard.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  List,
+  ListItem,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import IsoDate from 'foremanReact/components/common/dates/IsoDate';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+
+export const RegisteredBy = ({ user, activationKey }) => {
+  if (user) {
+    return (
+      <DescriptionListDescription>{user}</DescriptionListDescription>
+    );
+  }
+  return (
+    <>
+      <List isPlain>
+        {activationKey.map(key => (
+          <ListItem key={key.id}>
+            <Text component={TextVariants.h4}>Activation key</Text>
+            <a href={urlBuilder(`activation_keys/${key.id}`, '')}>{key.name}</a>
+          </ListItem>
+        ))}
+      </List>
+    </>
+  );
+};
+
+RegisteredBy.propTypes = {
+  user: PropTypes.string,
+  activationKey: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.number,
+    name: PropTypes.string,
+  })),
+};
+
+RegisteredBy.defaultProps = {
+  user: '',
+  activationKey: {},
+};
+
+const RegistrationCard = ({ isExpandedGlobal, hostDetails }) => {
+  /* eslint-disable max-len */
```

Comment:
seems like this is no longer necessary

Reply:
I added it because it is upset about this line:

`const subscriptionFacetAttributes = propsToCamelCase(hostDetails?.subscription_facet_attributes || {});` it's 6 over the 100 allowed limit. I have an enable after that to turn back on linting